catalog/spatial/csw/spatial-csw-source/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/source/CswSource.java

```diff
@@ -438,8 +452,7 @@ public void refresh(Map<String, Object> configuration) throws SecurityServiceExc
             cswSourceConfiguration.setIdentifierMapping(identifierMapping);
         }
 
-        Boolean sslProp = (Boolean) configuration
-                .get(DISABLE_CN_CHECK_PROPERTY);
+        Boolean sslProp = (Boolean) configuration.get(DISABLE_CN_CHECK_PROPERTY);
```

Comment:
If this is a checkbox in the metatype, we've had problems with those sometimes coming back as a String instead of a Boolean. This would then blow up with a class cast exception. It is safer to do `Boolean.parseBoolean(String.valueOf(configuration.get(DISABLE_CN_CHECK_PROPERTY)))`